response_operations_ui/controllers/respondent_controllers.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def search_respondent_by_email(email):
+    logger.debug('Search respondent via email')
+
+    request_json = {
+        'email': email
+    }
+    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/party/get-respondent-by-email'
+    response = requests.get(url, json=request_json)
+
+    if 'No respondent found' in response:
```

Comment:
Will the response also be a 404? It's probably better to check on the status code rather than text in the response

Reply:
This is being returned from the API as a 200. The response text is the only difference. I don't know why but we only seem to handle 200s and 500s in this repo and backstage.